incubator/java-openliberty/templates/default/src/main/liberty/config/server.xml

```diff
@@ -3,6 +3,8 @@
         <feature>microProfile-3.2</feature>
     </featureManager>
 
+    <quickStartSecurity userName="${stack.username}" userPassword="${stack.password}" />
```

Comment:
It looks great from the perspective of something new, but we changed the "contract" on configDropins/defaults mid-stream.   

Is this OK?   Can you maybe spell out some more thought about why this shouldn't be a concern?   Is there any way to do this with a file named:  `configDropins/defaults/quick-start-security.xml`?

Reply:
Not sure what you mean by "contract."  Is the concern that we had a file called quick-start-security.xml and now we dont? Or that we were removing it and now we aren't? Given that this is just a change to the template, not sure if it matters.